README.md

```diff
@@ -6,9 +6,12 @@ The Hot Potato is an ASP.NET Core reverse proxy that will validate an API's conf
 
 ## Setup
 
-To use the complete tool you will need to download the `HotPotato.AspNetCore.Host` NuGet package from https://www.nuget.org/packages/HotPotato.AspNetCore.Host/. Since Hot Potato is a dotnet global tool you can easily download it from Powershell or Command Prompt.
+To use the complete tool you will need to download the `HotPotato.AspNetCore.Host` NuGet package from https://www.nuget.org/packages/HotPotato.AspNetCore.Host/. Since Hot Potato is a dotnet global tool you can easily download it from Powershell or Command Prompt. The most common way of utilizing this tool can be found in the [Postman](#postman) section below.
```

Comment:
Since this is near the top, we may want to include the `dotnet tool install` command up here so people can just quickly copy/paste.

Reply:
Nevermind, I see it's just below this.